client/app/dbaas/logs/logs-constants.js

```diff
@@ -170,10 +170,10 @@ angular.module('managerApp').constant('LogsConstants', {
       COLD_STORAGE_TOTAL: 'ldp.service.coldstorage.total',
     },
     AGGREGATORS: {
-      MAX: 'max',
+      ZIMSUM: 'zimsum',
```

Comment:
Could you please explain in a comment what is a `ZIMSUM` please ? :)

Reply:
http://opentsdb.net/docs/build/html/user_guide/query/aggregators.html#zimsum

In our case, it makes our chart much more realistic.